wolfcrypt/src/wc_pkcs11.c

```diff
@@ -843,9 +977,24 @@ static int Pkcs11CreateEccPrivateKey(CK_OBJECT_HANDLE* privateKey,
         { CKA_KEY_TYPE,  &ecKeyType,    sizeof(ecKeyType)    },
         { operation,     &ckTrue,       sizeof(ckTrue)       },
         { CKA_EC_PARAMS, params,        0                    },
-        { CKA_VALUE,     NULL,          0                    }
+        { CKA_VALUE,     NULL,          0                    },
+        { 0,             NULL,          0                    },
+        { 0,             NULL,          0                    }
     };
-    CK_ULONG        keyTmplCnt = sizeof(keyTemplate) / sizeof(*keyTemplate);
+    CK_ULONG        keyTmplCnt = sizeof(keyTemplate) / sizeof(*keyTemplate) - 2;
```

Comment:
Can you clarify why the empty entries above are required and the `-2`?

Reply:
Comments added - optional entries.